iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/voter/AffOrgMatchVotersFactory.java

```diff
@@ -15,43 +15,62 @@ private AffOrgMatchVotersFactory() { }
     
     //------------------------ LOGIC --------------------------
     
-    public static AffOrgMatchVoter createNameCountryStrictMatchVoter() {
+    public static AffOrgMatchVoter createNameCountryStrictMatchVoter(float matchStrength) {
         
-        return new CompositeMatchVoter(ImmutableList.of(
+        CompositeMatchVoter voter = new CompositeMatchVoter(ImmutableList.of(
                 new CountryCodeStrictMatchVoter(), 
                 new NameStrictWithCharFilteringMatchVoter(ImmutableList.of(',', ';'))));
         
+        voter.setMatchStrength(matchStrength);
```

Comment:
Is there any scenario we wouldn't like to set match strenght to `CompositeMatchVoter`? If no, maybe we could enforce providing it as constructor parameter?

Related to this line and all similar lines above.